### PR TITLE
Added travis config.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by bshaffer** [comment]
I wouldn't mind formatting changes being in a separate PR, in the future

File: testing/variables.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    GOOGLE_STORAGE_BUCKET
+    MYSQL_DSN
+    MYSQL_USER
+    MYSQL_PASSWORD
+    GOOGLE_CREDENTIALS_BASE64
+)
+
+DIRS=(
+    ${TRAVIS_BUILD_DIR}/helloworld/silex
+    ${TRAVIS_BUILD_DIR}/bookshelf
```
Comment (bshaffer) [inline testing/variables.sh]: so does this mean running `testing/run_test_suite.sh` will only work on travis?

Comment (tmatsuo) [inline testing/variables.sh]: Changed it to more generic TEST_BUILD_DIR

**Comment by tmatsuo** [comment]
@bshaffer Yeah, it makes sense. I'll separate them next time.

**Comment by tmatsuo** [comment]
Finally all the tests pass :)
@bshaffer I appreciate it if you can take a look.

File: testing/variables.sh (hunk 1 above)
Comment (tmatsuo) [inline testing/variables.sh]: Very good point, thanks!

File: testing/dump_credentials.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Dumps the contents of the environment variable GOOGLE_CREDENTIALS_BASE64 to
+ * a file.
+ *
+ * To setup Travis to run on your fork, read TRAVIS.md.
```
Comment (bshaffer) [inline testing/dump_credentials.php]: we should remove this line or add a `TRAVIS.md`

Comment (tmatsuo) [inline testing/dump_credentials.php]: Good point. I added the file.

File: testing/install_test_dependencies.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+done
+
+# Exit when any of the necessary envvar is not set.
+if [ "${PREREQ}" = "false" ]; then
+    exit 1
+fi
+
+# Install composer
+mkdir -p ${HOME}/bin
+curl -sS https://getcomposer.org/installer | php -- --install-dir=${HOME}/bin --filename=composer
```
Comment (bshaffer) [inline testing/install_test_dependencies.sh]: awesome, I didn't know you could send those arguments in! Those are handy.

File: testing/variables.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Defines some variables
+
+VARS=(
+    CLOUDSDK_ACTIVE_CONFIG_NAME
```
Comment (bshaffer) [inline testing/variables.sh]: Where is this env var being used? Do we still need it?

Comment (tmatsuo) [inline testing/variables.sh]: This is used by the gcloud command. It's safe to use for isolating the gcloud configs.

Comment (tmatsuo) [inline testing/variables.sh]: Oh, and it's not used for now, but I will use it when I add the e2e test.

**Comment by bshaffer** [comment]
LGTM

**Comment by tmatsuo** [comment]
Thx!

**Pull request merged**